modules/osd-monitoring-exploring-the-visualized-metrics.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-managing-metrics.adoc
+
+[id="exploring-the-visualized-metrics_{context}"]
+= Exploring the visualized metrics
+
+After running the queries, the metrics are displayed on an interactive plot. The X-axis in the plot represents time and the Y-axis represents metrics values. Each metric is shown as a colored line on the graph. You can manipulate the plot interactively and explore the metrics.
+
+.Procedure
+
+In the *Administrator* perspective:
```

Comment:
I don;t have access to a deployment either, but i assume so because SRE did not delete it from the Google doc they gae me. But let's make sure.

 @sichvoge Can a `dedicated-admin` in OSD and ROSA access the _Administrator Perspective_?

Reply:
Yes. (I check with Sam and Taylor in a Slack chat.)